apis/r/R/SOMAExperimentAxisQuery.R

```diff
@@ -123,8 +123,10 @@ SOMAExperimentAxisQuery <- R6::R6Class(
     #' @param X_layers The name(s) of the `X` layer(s) to read and return.
     #' @param obs_column_names,var_column_names Specify which column names in
     #' `var` and `obs` dataframes to read and return.
+    #' @param n_cores Number of cores to use for parallelized reads; disabled
+    #' on Windows
     read = function(
-      X_layers = NULL, obs_column_names = NULL, var_column_names = NULL) {
+      X_layers = NULL, obs_column_names = NULL, var_column_names = NULL, n_cores = 1L) {
```

Comment:
There is an _existing_ option R use for this, we use this in tiledb-R as well (mostly to throttle _down_ at CRAN at their request, please see https://github.com/TileDB-Inc/TileDB-R/blob/master/R/Config.R#L226-L234  

So maybe using `n_cores=getOption("Ncpus", 2L)` (or 1) is what we want here?

Reply:
My philosophy is to default to no parallelization (1), and require that users who want to parallelization should enable it explicitly

Reply:
CRAN allows two in all settings.  Many users want more.

But TileDB Core uses all the cores it can get (unless the user overrides).  So does TileDB-R.